src/gui/folderman.cpp

```diff
@@ -1456,6 +1452,11 @@ Folder *FolderMan::addFolderFromWizard(AccountStatePtr accountStatePtr, const QS
     return newFolder;
 }
 
+Folder *FolderMan::addFolderFromWizardResult(AccountStatePtr accountStatePtr, const FolderWizard::Result &config)
```

Comment:
The name is ambiguous since we have two wizards.

Reply:
And both functions can be called from anywhere

Reply:
The problem is that the result you pass is not a "wizard result", it is a "folder wizard result". In case you want to generalize the result class, it should be moved to another namespace.

Reply:
P.S.: We should try to always distinguish properly between folder wizard and setup wizard to avoid any confusion within the code base.

Reply:
Not sure whether the now ridiculously long name helps but here it is.